### PR TITLE
[NO-ISSUE] Added eslint and commit-branch-modifier

### PR DESCRIPTION
**Pull request by Epitrochoid**
The indentation level change is going to create a painful merge conflict, but not much that I can do about it.

### CODE REVIEW

File: package.json
Comment (ctdio) [inline package.json]: `eslint .` would be better, since we should be enforcing our standards on all files (including tests).

**Review by ctdio — CHANGES_REQUESTED** [review]

**Review by Druotic — APPROVED** [review]
Other than the lint script change (including all root dirs/files), looks good to me. Approving with the expectation that the linting change will happen before merging.

**Pull request merged**
